### DISCUSSION TITLE
The ability to send notifications to people based on a filter when an item is created

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#11677] [Feature Requests]
The ability through settings in the data model to create a notification on the creation of an item defined by a filter.

Example - On the creation of an item in the posts section create a notification for all users in the group.

Post -> Group -> Users

Not too sure how this would be done but might be possible

**Comment by rijkvanzanten** [comment]
Love it! As a matter of fact, I'm happy to report that we're currently working on a new feature that would allow for this 🙂

**Comment by rijkvanzanten** [comment]
This is now possible through the use of Flows!
